kotlin-native/NEW_MM.md

```diff
@@ -78,40 +72,40 @@ If `kotlin.native.isExperimentalMM()` returns `true`, you've successfully enable
 
 ### Update the libraries
 
-To fully take advantage of the new MM, newer versions of libraries were released:
+For you to take full advantage of the new MM, we released new versions of the following libraries:
 * `kotlinx.coroutines`: `1.5.1-new-mm-dev2` at https://maven.pkg.jetbrains.space/public/p/kotlinx-coroutines/maven
-  * No freezing, every common primitive (Channels, Flows, coroutines) work through worker boundaries.
-  * Unlike `native-mt` version, library objects are transparent for `freeze`: if you freeze e.g. a channel, all of its internals will get frozen, 
-    and it won't work as expected. In particular, this can happen when freezing something that captures a channel.
-  * `Dispatchers.Default` is backed by a pool of workers on Linux and Windows, and by a global queue on Apple targets.
-  * `newSingleThreadContext` to create coroutine dispatcher backed by a worker.
-  * `newFixedThreadPoolContext` to create coroutine dispatcher backed by a pool of `N` workers.
-  * `Dispatchers.Main` backed by main queue on Darwin and by standalone worker on other platforms. **NOTE**: _Don't use `Dispatchers.Main` in unit-tests, because nothing is processing the main thread queue in unit-tests._
+    * No freezing. Every common primitive (Channels, Flows, coroutines) works through `Worker` boundaries.
+    * Unlike the `native-mt` version, library objects are transparent for `freeze`. For example, if you freeze a channel, all of its internals will get frozen, so it won't work as expected. In particular, this can happen when freezing something that captures a channel.
+    * `Dispatchers.Default` is backed by a pool of `Worker`s on Linux and Windows and by a global queue on Apple targets.
+    * `newSingleThreadContext` to create a coroutine dispatcher backed by a `Worker`.
+    * `newFixedThreadPoolContext` to create a coroutine dispatcher backed by a pool of `N` `Worker`s.
+    * `Dispatchers.Main` is backed by the main queue on Darwin and by a standalone `Worker` on other platforms.
+      _Don't use `Dispatchers.Main` in unit-tests because nothing processes the main thread queue in unit-tests._
```

Comment:
usually, we write `unit tests` without the hyphen
Is it a special name?

Reply:
Not a special name, no.